### PR TITLE
Change the telegram icon in the social media icons

### LINKED ISSUE #166: Change the telegram icon in the social media icons
- Currently, the telegram icon inserted is wrong
- Add the new telegram icon `https://img.icons8.com/fluency/48/telegram-app.png`

image

### PR DESCRIPTION
**Pull request by maddy020**
## Purpose
Icon of telegram is wrong and it need to be replaced with other link

The purpose of this Pull Request is to fix #166 <issue-number>

## Goals
Icon of telegram is wrong and it need to be replaced with other link

## Approach
Simply change the link of source image

### Before
before

### After
after

### Preview Link

https://pr-171-NEW-YOSHITHA-WEBSITE.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: dark/index.html [hunk 1]
```diff
@@ -150,7 +150,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">Telegram</span>
-        <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
+        <a href="https://img.icons8.com/fluency/48/telegram-app.png" target="_blank "><img
```
Comment (YoshithaRathnayake) [inline dark/index.html:153]: You have changed the icon link @maddy020, Don't remove that undo that and change the image source link.
```suggestion
        <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
```

File: dark/index.html [hunk 2]
```diff
@@ -150,7 +150,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">Telegram</span>
-        <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
+        <a href="https://img.icons8.com/fluency/48/telegram-app.png" target="_blank "><img
             src="https://img.icons8.com/color/48/000000/telegram-app--v1.png" alt="Telegram Icon" /></a>
```
Comment (YoshithaRathnayake) [inline dark/index.html:154]: Change this to following @maddy020 

```suggestion
            src="https://img.icons8.com/fluency/48/telegram-app.png" alt="Telegram Icon" /></a>
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by maddy020** [comment]
Please review it and help in merging the PR

**Comment by maddy020** [comment]
Has done the above changes

**Review by YoshithaRathnayake — APPROVED** [review]
Thank You! @maddy020

**Pull request merged**